### ISSUE TITLE
Rename (GET) /api/v1/aquarius/assets/ddo/query to (GET) /api/v1/aquarius/assets/ddo/search

### ISSUE THREAD

**Issue by innoprenuer** [oceanprotocol/aquarius#117]
#### Description

- Currently, for search/query function we are exposing 2 similar endloints with only difference being the method (GET and POST).
- For **GET** -
`/api/v1/aquarius/assets/ddo/query` (description - Get a list of DDOs that match with the given text)
As per descriptin, this endpoint seaches based on **text** input (simple text search). So, ideally it shuould be renamed as `/api/v1/aquarius/assets/ddo/search` or `/api/v1/aquarius/assets/ddo/text`.
- All advanced model based queries will be handled using other endpoint (POST) `/api/v1/aquarius/assets/ddo/query`

**Comment by ttmc** [comment]
A more RESTful form would be:

`GET /api/v1/aquarius/assets/ddos?text_search={text}`

The stuff after the `?` is called an HTTP "query string" for a reason: everything before the `?` gets you the whole list of DDOs, and then the stuff after the `?` does a search within that list.

Also, `aquarius/assets/` is unnecessary and could be removed too, but that's a matter for a different issue, since it affects all the other endpoints as well.

**Comment by ttmc** [comment]
On second thought, why are we putting _any_ search or query endpoints in the Aquarius REST API? It will take developer time to write and maintain that code (for all possible backend databases) and for what? Marketplace developers who use Aquarius can just connect directly to the backend database (e.g. MongoDB, Elasticsearch or whatever) and use the already-existing powerful SDKs that come with those databases (e.g. pymongo) to do ultra-sophisticated queries.

In other words, let's not reinvent the wheel.

I'm happy to discuss this offline if you like.
